content/release-notes/2023-06-13-storage-and-compute.md

```diff
@@ -0,0 +1,10 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Added support for the `pgx_ulid` extension, which enables  using ULIDs within PostgreSQL.
```

Comment:
extra space between "enables" and "using"

Reply:
Thank you. Fixed.